modules/ob-installing-s2i-strategy.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-s2i-strategy_{context}"]
+= Installing source-to-image strategy
+
+You can install the `source-to-image` strategy at cluster level. By installing the `source-to-image` strategy at cluster level, you can share it across different namespaces within your cluster.
```

Comment:
Same comment as above: "You can share the `source-to-image` strategy across different namespaces within your cluster by installing it at cluster level.